packages/mongodb-chatbot-server/src/routes/content/contentRouter.test.ts

```diff
@@ -1,6 +1,103 @@
-// TODO: test the router
+import request from "supertest";
+import { makeTestApp } from "../../test/testHelpers";
+import type { MakeContentRouterParams } from "./contentRouter";
+import type { SearchResultsStore, SearchResultRecord } from "mongodb-rag-core";
+
+// Minimal in-memory mock for SearchResultsStore for testing purposes
+const mockSearchResultsStore: SearchResultsStore = {
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  drop: async () => {},
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  close: async () => {},
+  metadata: {
+    databaseName: "mock",
+    collectionName: "mock",
+  },
+  async saveSearchResult(_record: SearchResultRecord) {
+    // Not implemented
+  },
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  async init() {},
+};
+
+// Helper to build contentRouterConfig for the test app
+function makeContentRouterConfig(
+  overrides: Partial<MakeContentRouterParams> = {}
+) {
+  return {
+    findContent: jest
+      .fn()
+      .mockResolvedValue({ content: [], queryEmbedding: [] }),
+    searchResultsStore: mockSearchResultsStore,
+    ...overrides,
+  };
+}
+
 describe("contentRouter", () => {
-  it("should create a new conversation", () => {
-    expect(true).toBe(true);
+  const searchEndpoint = "/api/v1/content/search";
+
+  it("should return search results for a valid request", async () => {
+    const { app, origin } = await makeTestApp({
+      contentRouterConfig: makeContentRouterConfig(),
+    });
+    const res = await request(app)
+      .post(searchEndpoint)
+      .set("req-id", "test-req-id")
+      .set("Origin", origin)
+      .send({
+        query: "mongodb",
+        limit: 2,
+      });
```

Comment:
to make the code more DRY, can you create a new helper function called `makeSearchContentRequest(params: { body: SearchContentRequestBody))` that you use across the tests.